smart-employee-agent/client/serve.py

```
@@ -0,0 +1,75 @@
+        # SPA routing: non-file paths serve index.html
+        file_path = DIRECTORY / path.lstrip("/")
+        if not file_path.is_file() and not path.startswith("/api"):
+            self.path = "/index.html"
+
+        return super().do_GET()
+
+    def end_headers(self):
+        self.send_header("Cache-Control", "no-store, no-cache, must-revalidate")
+        super().end_headers()
+
+    def log_message(self, format, *args):
```

Ruff (0.15.11)

[error] Function argument `format` is shadowing a Python builtin

(A002)